bridge/shared/models/trace.ts

```diff
@@ -228,7 +228,7 @@ export class Trace {
   public isFaulty(stageName?: string): boolean {
     let result = false;
     if (this.data) {
-      if (this.isFailed() || this.traces.some((t) => t.isFaulty())) {
+      if (this.isFailed()) {
```

Comment:
This may not work with the webhook.
Example
```
evaluationSequence.triggered
-> webhook.triggered
-> webhook.started
-> trigger delivery
---->start, finished
-> trigger evaluation
---->start, finished
-> webhook.finished
-> evaluationSequence.finished
```
for `isFailed`, it takes the result of the first found finished event.
If the webhook succeeds, the overall sequence is success but it may be failed.

Correct me if I'm wrong here

Reply:
You're right this doesn't work with webhook events. With help of @ermin-muratovic this should be fixed now. There is also a UI test that checks for it.